crates/op-rbuilder/README.md

```diff
@@ -77,34 +75,29 @@ git checkout op-rbuilder
 - Windows: `{FOLDERID_RoamingAppData}/reth/`
 - macOS: `$HOME/Library/Application Support/reth/`
 
-3. Run a clean OP stack in the `optimism` repo:
-
-```bash
-make devnet-clean && make devnet-down && make devnet-up
-```
-
-4. Run `op-rbuilder` in the `rbuilder` repo on port 8547:
+4. Run `op-rbuilder` in the `rbuilder` repo on port 4444:
 
 ```bash
-cargo run -p op-rbuilder --bin op-rbuilder --features optimism -- node \
-    --chain ../optimism/.devnet/genesis-l2.json \
-    --http \
-    --http.port 8547 \
-    --authrpc.jwtsecret ../optimism/ops-bedrock/test-jwt-secret.txt \
+cargo run -p op-rbuilder --bin op-rbuilder --features flashblocks -- node \
+    --chain $HOME/.playground/devnet/l2-genesis.json \
+    --http --http.port 2222 \
+    --authrpc.port 4444 --authrpc.jwtsecret $HOME/.playground/devnet/jwtsecret \
+    --port 30333 --disable-discovery \
     --metrics 127.0.0.1:9001 \
-    --rollup.builder-secret-key ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+    --rollup.builder-secret-key ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
+    --trusted-peers enode://3479db4d9217fb5d7a8ed4d61ac36e120b05d36c2eefb795dc42ff2e971f251a2315f5649ea1833271e020b9adc98d5db9973c7ed92d6b2f1f2223088c3d852f@127.0.0.1:30304
 ```
 
 5. Init `mev-flood`:
 
 ```bash
-docker run mevflood init -r http://host.docker.internal:8547 -s local.json
+docker run -v ${PWD}:/app/cli/deployments flashbots/mev-flood init -r http://host.docker.internal:2222 -s local.json
 ```
 
 6. Run `mev-flood`:
 
 ```bash
-docker run --init -v ${PWD}:/app/cli/deployments mevflood spam -p 3 -t 5 -r http://host.docker.internal:8547 -l local.json
+docker run --init -v ${PWD}:/app/cli/deployments flashbots/mev-flood spam -p 3 -t 5 -r http://host.docker.internal:2222 -l local.json
```

Comment:
For the mev-flood steps to work out of the box I believe we need the accounts to be prefunded (see https://github.com/flashbots/builder-playground/pull/103)

Reply:
instead of mev-flood we use contender now, wouldn't worry too much about the mev-flood docs. We'll also need to add the default anvil addresses into the prefunded list of addresses in playground cc @ferranbt

Reply:
Happy to update my PR to add the additional addresses in a minute

Reply:
Sounds great, happy to add the anvil docs to the PR above. I'll update this PR to use contender vs. mevflood